addons/storyshots/src/angular/renderTree.js

```diff
@@ -1,8 +1,10 @@
-/* eslint-disable import/no-extraneous-dependencies */
 import AngularSnapshotSerializer from 'jest-preset-angular/AngularSnapshotSerializer';
 import HTMLCommentSerializer from 'jest-preset-angular/HTMLCommentSerializer';
+// eslint-disable-next-line
```

Comment:
please disable only the specific rule (`// eslint-disable-next-line import/no-extraneous-dependencies`)